packages/completer/src/handler.ts

```diff
@@ -616,6 +629,8 @@ export namespace CompletionHandler {
     items: CompletionHandler.ICompletionItems;
   }
 
+  export const ICompletionItemsResponseType = 'ICompletionItemsReply';
```

Comment:
If we export this as a const type, then I believe when we do `typeof` below it will only allow this particular string.

```suggestion
  export const ICompletionItemsResponseType = 'ICompletionItemsReply' as const;
```